### ISSUE TITLE
feature: Update Placement Translator to concatenate individual yaml into edgeDeployment objects in KubeStellar namespace

### ISSUE THREAD

**Issue by clubanderson** [kubestellar/kubestellar#1130]
### Feature Description

with the addition of an edgeDeployment, it is necessary to update the Placement Translator to create edgeDeployment objects in the KubeStellar Namespace.  Content for the edgeDeployment comes from combining all yaml from objects in the WDS that are targeted for the location(s) in the singleplacementslice objects created by the Where Resolver.

### Proposed Solution

update the Placement Translator to create edgeDeployment objects in the KubeStellar namespace

 should include information about location(s) and labels from the edgePlacement object that help direct the actions of the KubeStellar syncer - such as "override" or other special behaviors.

### Want to contribute?

- [ ] I would like to work on this issue.

**Comment by nirrozenbaum** [comment]
@clubanderson this part is handled differently in the design we proposed to the team that was approved.
we suggested using a new CRD in WDS called EdgePlacementDecision, which includes all objects + all destinations, mapped to a single EdgePlacement.
then, a generic transport controller (which is already implemented) is watching this CR, bundling it into a single wrapped object, and putting the wrapped object in the mailbox namespace.
the only requirement from is to supply a `WrapObjects` function.
so this feature is going to be handled, but different than what is stated in the title.
